pyro/poutine/enumerate_poutine.py

```diff
@@ -0,0 +1,68 @@
+from __future__ import absolute_import, division, print_function
+
+from .poutine import Messenger, Poutine
+
+
+def _iter_discrete_filter(msg):
```

Comment:
this is defined in two places

Reply:
this = _iter_discrete_filter

Reply:
They differ, but I'll rename them to make that clear...